catalog/dags/common/constants.py

```diff
@@ -36,3 +36,4 @@
 )
 AWS_CONN_ID = os.getenv("AWS_CONN_ID", "aws_conn_id")
 AWS_RDS_CONN_ID = os.environ.get("AWS_RDS_CONN_ID", AWS_CONN_ID)
+ES_PROD_CONN_ID = "elasticsearch_production"
```

Comment:
Since we're moving away from the pattern of defining a separate `CONN_ID`, I did not do so here.